lib/js-api/src/api/content-rest-api/model/groupBodyUpdate.ts

```diff
@@ -17,4 +17,5 @@
 
 export interface GroupBodyUpdate {
     displayName: string;
+    description?: string;
```

Comment:
same for [GroupsApi.md](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/js-api/src/api/content-rest-api/docs/GroupsApi.md)

Reply:
done